### PR TITLE
Add cloudtrail_logging_disabled query Closes #1575

### LINKED ISSUE #1575: Add cloudtrail_logging_disabled query for Ansible
**Platform**
Ansible

**Provider**
AWS

**Description**
Checks if the logging has been enabled

### PR DESCRIPTION
**Pull request by NunoAraujoCX**
Closes #1575 
This query checks if the logging has been enabled

### CODE REVIEW

File: assets/queries/ansible/aws/cloudtrail_logging_disabled/query.rego [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package Cx
+
+CxPolicy[result] {
+    document := input.document[i]
+    tasks := getTasks(document)
+    task := tasks[t]
+    task["community.aws.cloudtrail"].enable_logging == false
```
Comment (joaoReigota1) [inline assets/queries/ansible/aws/cloudtrail_logging_disabled/query.rego:7]: Ansible also allows "false" and "no" we should check for these issues using 
isAnsibleFalse(answer) {
     lower(answer) == "no"
} else {
     lower(answer) == "false"
} else {
     answer == false
}

File: assets/queries/ansible/aws/cloudtrail_logging_disabled/metadata.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "cloudtrail_logging_disabled",
+  "queryName": "CloudTrail Logging Disabled",
+  "severity": "HIGH",
+  "category": "Logging",
+  "descriptionText": "Enables Logging for the trail",
```
Comment (joaoReigota1) [inline assets/queries/ansible/aws/cloudtrail_logging_disabled/metadata.json:6]: The query description should point out the vulnerability being address or how to fix it

**Review by joaoReigota1 — CHANGES_REQUESTED** [review]

**Comment by markmishaevcx** [comment]
Scan submitted to Checkmarx

**Comment by markmishaevcx** [comment]
Logo<br />**Checkmarx SAST - Scan Summary & Details**
### Cx-SAST Summary
Total of 4 vulnerabilities<br />High&nbsp;**0 High**<br />Medium&nbsp;**3 Medium**<br />Low&nbsp;**1 Low**<br />Info&nbsp;**0 Info**<br />
### Violation Summary
Medium&nbsp;**3** **Medium**<br />
[View more details on Checkmarx UI](https://sast.checkmarx.net/CxWebClient/ViewerMain.aspx?scanid=1049090&projectid=737)
### Cx-SAST Details
|Lines|Severity|Category|File|Link|
---|---|---|---|---
[30](https://github.com/Checkmarx/kics/blob/feature/ansible_enable_cloudtrail_logging/cmd/builder/main.go#L30) |Medium|Path_Traversal|cmd/builder/main.go|[Checkmarx](https://sast.checkmarx.net/CxWebClient/ViewerMain.aspx?scanid=1049090&projectid=737&pathid=3)
[83](https://github.com/Checkmarx/kics/blob/feature/ansible_enable_cloudtrail_logging/test/main_test.go#L83) |Medium|Denial_Of_Service_Resource_Exhaustion|test/main_test.go|[Checkmarx](https://sast.checkmarx.net/CxWebClient/ViewerMain.aspx?scanid=1049090&projectid=737&pathid=2)
[52](https://github.com/Checkmarx/kics/blob/feature/ansible_enable_cloudtrail_logging/pkg/kics/service.go#L52) |Medium|Denial_Of_Service_Resource_Exhaustion|pkg/kics/service.go|[Checkmarx](https://sast.checkmarx.net/CxWebClient/ViewerMain.aspx?scanid=1049090&projectid=737&pathid=1)

**Comment by markmishaevcx** [comment]
Scan not submitted to Checkmarx due to existing Active scan for the same project.

**Review by joaoReigota1 — APPROVED** [review]

**Pull request merged**